plonk/src/recursion/circuits/split_acc.rs

```diff
@@ -98,11 +98,6 @@ impl SplitAccumulationInfo {
         let mut transcript = outputs[0].transcript.clone();
         transcript.merge(&outputs[1].transcript)?;
 
-        accumulators.iter().try_for_each(|acc| {
```

Comment:
split accumulation in the decider circuit, see https://github.com/EYBlockchain/nightfish_CE/pull/89/files#r2321574137